playing_cards.py

```diff
@@ -0,0 +1,71 @@
+import random
+
+
+def create_playing_cards():
+    shapes = ["Spade", "Heart", "Diamond", "Club"]
+    playing_cards = []
+
+    for shape in shapes:
+        for i in range(1, 14):
+            playing_cards.append((shape, i))
```

Comment:
Use a comprehention instead -
https://www.geeksforgeeks.org/comprehensions-in-python/

Reply:
fixed it